cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -78,6 +105,34 @@ void ExportedStubFunctions::AddMappings(Engine& engine) const {
   engine.AddGlobalMappingForFunc("gdv_fn_to_date_utf8_utf8_int32",
                                  types.i64_type() /*return_type*/, args,
                                  reinterpret_cast<void*>(gdv_fn_to_date_utf8_utf8_int32));
+
+  // gdv_fn_in_expr_lookup_int32
+  args = {types.i64_type(),  // int64_t in holder ptr
```

Comment:
I'm curious: is it possible to declare those signatures in the same file as the function they describe? It would probably makes things easier to follow and maintain.

And/or perhaps have some kind of declarative registry mechanism? I expect more runtime helpers will be needed along the way...

Reply:
Agreed we will definitely have more helpers coming soon. We can move these to the individual helper files.

By declarative do you mean some kind of config that automatically registers a holder to this list of exported functions?

(also can we address this in a follow up PR, i will update 3489 that this needs to be fixed)

Reply:
I mean some kind of macro or construct that would allow those to be top-level declarations somewhere. Agreed about addressing that in a followup PR, though.